### PR TITLE
Add "Bad" Licenses #1899

### LINKED ISSUE #1899: Add new "bad" licenses
See https://github.com/ErikMcClure/bad-licenses by @ErikMcClure
This is a list of off licenses that we should detect

### PR DESCRIPTION
**Pull request by viragumathe5**
### Added Licenses
1. Passive-Aggressive License
2. Non-White-Heterosexual-Male License
3. Hot Potato License
4. Protected Free To Use Software License
5. YOLO License
6. Do What The Fuck You Want To
7. LHA License
8. Minecraft Mod Public License
9. Anyone But Richard Stallman License~~
10. Nice License

Fixes #1899 

### Tasks

* [x] Reviewed [contribution guidelines](https://github.com/nexB/scancode-toolkit/blob/develop/CONTRIBUTING.rst)
* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] Tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR
  Run [tests](https://scancode-toolkit.readthedocs.io/en/latest/contribute/contrib_dev.html#running-tests) locally to check for errors. 
* [x] Commits are in a uniquely-named feature branch and has no merge conflicts 📁

### CODE REVIEW

File: src/licensedcode/data/licenses/non-white-heterosexual-male.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+key: non-white-heterosexual-male
+name: Passive-Aggressive License
```
Comment (TG1999) [inline src/licensedcode/data/licenses/non-white-heterosexual-male.yml]: Please check name and short name :).

File: LICENSE.txt [hunk 2]
```diff
@@ -0,0 +1,748 @@
+A. HISTORY OF THE SOFTWARE
```
Comment (pombredanne) [inline LICENSE.txt]: I think you did not mean to add this license text here at the root, correct?

File: src/licensedcode/data/licenses/anyone-but-richard-stallman.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+key: anyone-but-richard-stallman
+name: Anyone But Richard Stallman License
+short_name: ABRMS License
+category: Proprietary
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/anyone-but-richard-stallman.yml]: Use ` Proprietary Free` instead

File: src/licensedcode/data/licenses/dbad.yml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+key: dbad
+name: Dont Be A Dick Public License
+short_name: dbad
+category: Public
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/dbad.yml]: The category should be ` Proprietary Free`

File: src/licensedcode/data/licenses/hot-potato.yml [hunk 5]
```diff
@@ -0,0 +1,6 @@
+key: hot-potato
+name: Hot Potato
+short_name: Hot Potato License
+category: Permissive.
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/hot-potato.yml]: The category should be ` Proprietary Free`

File: src/licensedcode/data/licenses/lha.yml [hunk 6]
```diff
@@ -0,0 +1,5 @@
+key: lha
+name: LHA
+short_name: lha
+category: Permissive
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/lha.yml]: The category should be ` Proprietary Free`

File: src/licensedcode/data/licenses/minecraft-mod.yml [hunk 7]
```diff
@@ -0,0 +1,5 @@
+key: minecraft-mod
+name: Minecraft Mod License
+short_name: Minecraft Mod License 
+category: Public
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/minecraft-mod.yml]: The category should be ` Proprietary Free`

File: src/licensedcode/data/licenses/nice.yml [hunk 8]
```diff
@@ -0,0 +1,5 @@
+key: nice
+name: Nice License
+short_name: Nice License
+category: Permissive
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/nice.yml]: The category should be ` Proprietary Free`

File: src/licensedcode/data/licenses/non-white-heterosexual-male.yml [hunk 9]
```diff
@@ -0,0 +1,5 @@
+key: non-white-heterosexual-male
+name: Non White Heterosexual Male
+short_name: Non White Heterosexual Male
+category: Proprietary
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/non-white-heterosexual-male.yml]: The category should be ` Proprietary Free`

File: src/licensedcode/data/licenses/passive-aggressive.yml [hunk 10]
```diff
@@ -0,0 +1,5 @@
+key: passive-aggressive
+name: Passive-Aggressive License
+short_name: Passive-Aggressive License
+category: Permissive
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/passive-aggressive.yml]: The category should be ` Proprietary Free`

File: src/licensedcode/data/licenses/protected-free-to-use-software.LICENSE [hunk 11]
```diff
@@ -0,0 +1,51 @@
+The P.F.T.U.S(Protected Free To Use Software) License
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/protected-free-to-use-software.LICENSE]: Could you us `pftus-1.1` as a key for this instead?

File: src/licensedcode/data/licenses/protected-free-to-use-software.yml [hunk 12]
```diff
@@ -0,0 +1,5 @@
+key: protected-free-to-use-software
+name: Protected Free To Use Software License
+short_name: PFTUS-license
+category: Proprietary
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/protected-free-to-use-software.yml]: The category should be ` Proprietary Free`

File: src/licensedcode/data/licenses/yolo.yml [hunk 13]
```diff
@@ -0,0 +1,4 @@
+key: yolo
+name: yolo License
+short_name: YOLO License
+category: Proprietary
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/yolo.yml]: The category should be ` Proprietary Free`

File: src/licensedcode/data/licenses/dbad.yml (hunk 4 above)
Comment (viragumathe5) [reply src/licensedcode/data/licenses/dbad.yml]: But its a public license still should I write there Proprietary Free?????

File: LICENSE.txt (hunk 2 above)
Comment (viragumathe5) [reply LICENSE.txt]: done

**Comment by pombredanne** [comment]
Thank you for the updates... can you also fix the DCO signoff in your commit messages?

**Comment by pombredanne** [comment]
@viragumathe5 Why did you close this PR?  we are almost there and it is looking good!

File: src/licensedcode/data/licenses/non-white-heterosexual-male.yml (hunk 1 above)
Comment (pombredanne) [reply src/licensedcode/data/licenses/non-white-heterosexual-male.yml]: @viragumathe5 ^

**Comment by viragumathe5** [comment]
@pombredanne I am sorry I just messed up with local files so I am making new PR I am sorry for this.

**Comment by viragumathe5** [comment]
@pombredanne would you please tell me what's wrong with this I tried doing some common changes but I didn't get what's wrong with this Please

**Comment by pombredanne** [comment]
@viragumathe5 I added a few refinements to your PR and rebased on the latest develop. I will merged when this is passing the tests.
Thanks you!

**Pull request merged**